### DISCUSSION TITLE
Possible Android MAUI Bug

### DISCUSSION THREAD

**Discussion by TPitman-mindfiretechnology** [shinyorg/shiny#1221] [Q&A]
I think I found a beta MAUI Android bug in the Shiny bluetooth client code, but wanted to post here first to make sure I am not doing something wrong.

I just did a pull of the latest master branch so I am on the latest development code.

In shiny/src/Shiny.BluetoothLE/Platforms/Android/Peripheral.cs on line 82, cfg is null if the passed in parameter is not of type AndroidConnectionConfig. In that case cfg.AutoConnect throws a null ref exception.

I am not sure, but I think line 82 should be:

cfg = new AndroidConnectionConfig(config.AutoConnect);

**Comment by aritchie** [comment]
Please just file the issues in the proper place in the future

If you think it is a bug, send a reproducible case.  If you know it is a bug, send a reproducible case or PR
